universal-login-react/test/core/services/BackupCodesService.ts

```diff
@@ -0,0 +1,62 @@
+import chai, {expect} from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import sinonChai from 'sinon-chai';
+import sinon from 'sinon';
+import {TEST_TRANSACTION_HASH, INITIAL_GAS_PARAMETERS} from '@universal-login/commons';
+import {BackupCodesService} from '../../../src/core/services/BackupCodesService';
+
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+describe('UNIT: BackupCodesService', () => {
+  const waitToBeSuccess = sinon.stub().resolves([350, 506, 372, 483, 576, 48]);
```

Comment:
Looks like a type mismatch here. In the service the result of this function is `string[]`, here its `number[]`

Reply:
✔️